### ISSUE TITLE
Lesson 8: "Convolution Neural Networks" L30: "Convolutional Network in TensorFlow": error/typo

### ISSUE THREAD

**Issue by SherylHohman** [udacity/sdc-issue-reports#547]
`For example, the first layer shapes the images from 28x28x1 to 28x28x32 in the convolution step. Then next step applies max pooling, turning each sample into 14x14x32. `

Should read
`For example, the first layer shapes the images from 28x28x1 to 14x14x32 in the convolution step. Then next step applies max pooling, turning each sample into 7*7*64. `

Either that or the comments in the code is wrong, and should be adjusted to match the text.
def conv_net(x, weights, biases, dropout):
```
    # Layer 1 - 28*28*1 to 14*14*32
    conv1 = conv2d(x, weights['wc1'], biases['bc1'])
    conv1 = maxpool2d(conv1, k=2)

    # Layer 2 - 14*14*32 to 7*7*64
    conv2 = conv2d(conv1, weights['wc2'], biases['bc2'])
    conv2 = maxpool2d(conv2, k=2)

    # Fully connected layer - 7*7*64 to 1024
    fc1 = tf.reshape(conv2, [-1, weights['wd1'].get_shape().as_list()[0]])
    fc1 = tf.add(tf.matmul(fc1, weights['wd1']), biases['bd1'])
    fc1 = tf.nn.relu(fc1)
    fc1 = tf.nn.dropout(fc1, dropout)

    # Output Layer - class prediction - 1024 to 10
    ...

```

**Comment by swwelch** [comment]
Hi @SherylHohman, I think this looks correct to me. The code:
`
conv1 = conv2d(x, weights['wc1'], biases['bc1'])
`
has input dimensions 28x28x1 and output dimensions 28x28x32, and then the code:
`
conv1 = maxpool2d(conv1, k=2)
`
has input dimensions 28x28x32 with output dimensions 14x14x32. Similarly,
`
conv2 = conv2d(conv1, weights['wc2'], biases['bc2'])
conv2 = maxpool2d(conv2, k=2)
`
Goes from 14x14x32 to 7x7x64. It seems like text is just referring to the conv2d/maxpool2d step for `conv1`
